ui/src/features/project/project-details/nodes/repo-node.tsx

```diff
@@ -45,10 +47,10 @@ const RepoNodeBody = ({ label, value }: { label: string; value: string }) => (
   <>
     <div className={styles.label}>{label}</div>
     <Tooltip title={value}>
-      <div className={styles.value}>
+      <a href={urlWithProtocol(value)} className={styles.value} target='_blanl' rel='noreferrer'>
```

Comment:
Should that be `target='_blank'`  instead of `target='blanl'` ?

Reply:
good catch!